tests/test_settings.py

```diff
@@ -35,7 +39,8 @@ def test_defaults(self, mock_getcwd):
     @mock.patch('semantic_release.settings.getcwd', return_value='/tmp/')
     def test_toml_override(self, mock_getcwd):
         # create temporary toml config file
-        dummy_conf_path = '/tmp/pyproject.toml'
+        dummy_conf_path = os.path.join(here, 'tmp', 'pyproject.toml')
```

Comment:
This change is causing `True` to be returned for `upload_to_pypi` when it should be `False`.

Reply:
The problem is that on Windows, the path isn't being recognized if it contains `/`. The file does exist and has the proper content as stated on the next step but for some reason it isn't properly loaded on the assertion.

Reply:
Do these changes make the test pass on Windows? If so, we could add an OS-based switch using something like `platform.system()`.

Reply:
These two don't pass but due to the `AssertionError`. Before this change, Windows wasn't able to find the file path as the path separator had to be `\\` instead of `/`. Hence, throwing an error at the `pathlib` script, which is inside of pytest package.

Reply:
Aaah sorry, that's what's causing it on linux too. In that case we need to find what else is causing the `AssertionError`s.

Reply:
`config.getboolean('semantic_release', 'upload_to_pypi')` is returned as `True` but the `pyproject.toml` file has it as false value. 🤔